temp_test_e2e.py

```diff
@@ -0,0 +1,163 @@
+        "pytorch_version": "1.7.0",
+        "cuda_version": "11.0",
+        "cudnn_version": "cudnn8-devel",
+    }
+
+    mb = ModelBundle("abc123")
+
+    # TODO out of date lol
+    args = {
+        "service_name": "seantest",
+        "env_params": env_params,
+        # "bundle_id": "abc123",
+        "model_bundle": mb,
+        "cpus": 1,
+        "memory": "4Gi",
+        "gpus": 1,
+        "gpu_type": "nvidia-tesla-t4",
+        "min_workers": 1,
+        "max_workers": 1,
+        "per_worker": 1,
+        "requirements": [],
+    }
+
+    model_endpoint = hmi_client.create_model_endpoint(**args)
+    print(model_endpoint.endpoint_id)
+
+
+def make_task_call(
+    endpoint_name: str, dataset_id: str, upload_to_nucleus=True
+):
+    print("Need to export NUCLEUS_API_KEY=live_<your live api key here>")
+    client = nucleus.NucleusClient(os.environ["NUCLEUS_API_KEY"])
+
+    dataset = client.get_dataset(dataset_id)
+
+    model_endpoint = ModelEndpoint(
+        endpoint_id=endpoint_name, client=hmi_client
+    )
+
+    inference_run = create_nucleus_dataset_inference_run(
+        model_endpoint, client, dataset
+    )
+
+    while not inference_run.is_done(poll=True):
+        print("Waiting for predictions to finish...")
+        time.sleep(5)
+
+    print("Predictions complete!")
+    predictions = inference_run.hmi_async_job.get_responses()
+    print(predictions)
+
+    if upload_to_nucleus:
+        ts = str(time.time())
+        inference_run.upload_to_nucleus(
+            model_run_name=ts,
+            model_name="Test HMI upload",
+            model_ref_id=f"test_hmi_upload_{ts}",
+        )
+
+
+def temp_clone_pandaset():
+    print("Need to export NUCLEUS_API_KEY=live_<your live api key here>")
+    public_dataset_client = nucleus.NucleusClient(
+        os.environ["NUCLEUS_PUBLIC_DATASET_API_KEY"]
+    )
+
+    dataset = public_dataset_client.get_dataset(
+        "ds_bwhjbyfb8mjj0ykagxf0"
+    )  # Public Pandaset Dataset id
+    dataset_items = dataset.items
+
+    client = nucleus.NucleusClient(os.environ["NUCLEUS_API_KEY"])
+    cloned_dataset = client.create_dataset("Pandaset clone - small")
+    cloned_dataset_items = [
+        nucleus.DatasetItem(
+            image_location=x.image_location,
+            reference_id=x.reference_id,
+            metadata=x.metadata,
+        )
+        for x in dataset_items[:1]
+    ]
+    cloned_dataset.append(cloned_dataset_items)
+
+
+# temp_clone_pandaset()
+
+if __name__ == "__main__":
+    # make_task_call(
+    #     endpoint_name="yi-tf-test",
+    #     #dataset_id="ds_c4wht080x81g060m0nfg",
+    #     dataset_id="ds_c4x8s3m6n260060cngs0",
+    #     upload_to_nucleus=True
+    # )
+    # temp_clone_pandaset()
+
+    hmi_client = HostedInference(
+        api_key="live_38aa469bc8ab436d89dce6e37e8d5258"
+    )
+    img_url = "s3://scale-ml-hosted-model-inference/tmp/hosted-model-inference-outputs/c3f3b5ed-f182-4fa1-bfa5-9b2e017feb74.pkl"
```

Comment:
Probably best to remove any internal URLs in here. It's not the end of the world since's temporary and not a secret.